examples/ErrorHandling/HandlePartialFailures.cs

```diff
@@ -0,0 +1,186 @@
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Ads.GoogleAds.Lib;
+using Google.Ads.GoogleAds.V1.Common;
+using Google.Ads.GoogleAds.V1.Errors;
+using Google.Ads.GoogleAds.V1.Resources;
+using Google.Ads.GoogleAds.V1.Services;
+using System;
+using System.Collections.Generic;
+using static Google.Ads.GoogleAds.V1.Enums.AdGroupCriterionStatusEnum.Types;
+using static Google.Ads.GoogleAds.V1.Enums.KeywordMatchTypeEnum.Types;
+using static Google.Ads.GoogleAds.V1.Errors.ErrorLocation.Types;
+
+namespace Google.Ads.GoogleAds.Examples.V1
+{
+    /// <summary>
+    /// This code example demonstrates how to handle partial failures.
+    /// </summary>
+    public class HandlePartialFailures : ExampleBase
+    {
+        /// <summary>
+        /// Main method, to run this code example as a standalone application.
+        /// </summary>
+        /// <param name="args">The command line arguments.</param>
+        public static void Main(string[] args)
+        {
+            HandlePartialFailures codeExample = new HandlePartialFailures();
+            Console.WriteLine(codeExample.Description);
+            try
+            {
+                long adGroupId = long.Parse("INSERT_ADGROUP_ID_HERE");
+
+                // The Google Ads customer ID for which the call is made.
+                long customerId = long.Parse("INSERT_CUSTOMER_ID_HERE");
+
+                codeExample.Run(new GoogleAdsClient(), customerId, adGroupId);
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine("An exception occurred while running this code example. {0}",
+                    ExampleUtilities.FormatException(e));
+            }
+        }
+
+        /// <summary>
+        /// Returns a description about the code example.
+        /// </summary>
+        public override string Description
+        {
+            get
+            {
+                return "This code example demonstrates how to handle partial failures.";
+            }
+        }
+
+        /// <summary>
+        /// Runs the code example.
+        /// </summary>
+        /// <param name="client">The Google Ads client.</param>
+        /// <param name="customerId">The customer ID for which the call is made.</param>
+        /// <param name="adGroupId">Id of the ad group to which keywords are added.</param>
+        public void Run(GoogleAdsClient client, long customerId, long adGroupId)
+        {
+            // Get the AdGroupCriterionService.
+            AdGroupCriterionServiceClient adGroupCriterionService =
+                client.GetService(Services.V1.AdGroupCriterionService);
+
+            // Create the keywords. The second and fourth keywords have policy violation, and will
+            // fail with errors.
+            string[] keywords = new string[]
```

Comment:
Could you point me to other examples please?

Reply:
Currently there's one for Java and Python, the Python example is cloned from Java.

Java: https://github.com/googleads/google-ads-java/blob/5a2545b717a46c800f53bb8e3c8c870f08261d59/google-ads-examples/src/main/java/com/google/ads/googleads/examples/basicoperations/HandlePartialFailure.java

Python: https://team-review.git.corp.google.com/c/ads-api-devrel/google-ads-python/+/425673/

Reply:
+1 to making the examples consistent. This will be particularly important if we want to integrate examples into documentation.

@AnashOommen , could you align this one with the Java and Python examples?